CMakeLists.txt

```diff
@@ -83,7 +83,9 @@ spirv_cross_add_library(spirv-cross-core spirv_cross_core STATIC
     ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cross.hpp
     ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cross.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cfg.hpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cfg.cpp)
+    ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cfg.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/spirv_cross_util.hpp
```

Comment:
This should be a separate CMake target. spirv-cross-util or something and the "main" executable target should link to it.

Reply:
I'm not seeing an update Makefile?

Reply:
Sure, I can add the separate target. I'm not sure what I need to do for your second comment though, it seems like the Makefile should be ok?

Reply:
Actually, the Makefile is fine. It uses globbing.